### PR TITLE
Add button platform to moehlenhoff alpha 2

### PR DESCRIPTION
**Pull request by TitanNano**
## Proposed change

PR https://github.com/home-assistant/core/pull/85676 adds the button platform to the moehlenhoff_alpha2 integration and provides a virtual button that enables users to push their local home assistant time to the base stations internal clock.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/85676

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

File: source/_integrations/moehlenhoff_alpha2.markdown [hunk 1]
```diff
@@ -13,6 +13,7 @@ ha_platforms:
   - binary_sensor
```
Comment (MartinHjelmare) [inline source/_integrations/moehlenhoff_alpha2.markdown:13]:
```suggestion
  - binary_sensor
  - button
```

File: source/_integrations/moehlenhoff_alpha2.markdown [hunk 2]
```diff
@@ -13,6 +13,7 @@ ha_platforms:
...
   - climate
   - sensor
+  - button
```
Comment (MartinHjelmare) [inline source/_integrations/moehlenhoff_alpha2.markdown:16]:
```suggestion
```

**Comment by MartinHjelmare** [comment]
Too slow...

**Comment by frenck** [comment]
@MartinHjelmare 👍 
No problem, will be sorted on release / sync.

**Pull request merged**